### PR TITLE
feat: Addition of landing page for Mobile Application

### LINKED ISSUE #2929: Create an app section for the Attendee and Organizer app on the front page and mobile app pages
Please create sections for the mobile attendee app and organizer app on the start page.

Also create a landing page for the apps:
a) at /mobile-app for the attendee app (e.g. compare to https://www.eventbrite.com/eventbriteapp/) and b) at /organizer-app for the organizer app (e.g. compare to https://www.eventbrite.com/l/Organizer-check-in-app/)

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #2929 

#### Changes proposed in this pull request:

- Added `Attendee.hbs`
- Added `Organizer.hbs`
- Added images related to pages

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@shreyanshdwivedi @uds5501 Have a look and please check the pages

File: app/templates/attendee-app.hbs [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+          {{t 'hasslefree , easily and conveniently'}}
+        </h3>
+      </div>
+    </div>
+    <div class="eight wide column">
+      <img class="ui large image" src="/images/android-img.jpg" alt="Android Pic">
+    </div>
+  </div>
+</div>
+<div class="ui center aligned muted text">
```
Comment (uds5501) [inline app/templates/attendee-app.hbs]: add "header" to the class name in the end, to make it similar to organizer landing page

Comment (kushthedude) [reply app/templates/attendee-app.hbs]: @uds5501 Didnt notice the icons were collapsing in mobile view ,  👍

Comment (kushthedude) [reply app/templates/attendee-app.hbs]: @uds5501 Have a look now

Comment (uds5501) [reply app/templates/attendee-app.hbs]: @kushthedude 
- Decrease the icon size on mobile view
Screenshot from 2019-05-20 22-19-47

- Improve alignment of these paragraphs? They look odd.
Screenshot from 2019-05-20 22-20-14

Comment (kushthedude) [reply app/templates/attendee-app.hbs]: @uds5501 Check them now

**Comment by kushthedude** [comment]
@shreyanshdwivedi @uds5501 @pradeepgangwar Please check it now

File: app/templates/attendee-app.hbs (hunk 1 above)
Comment (uds5501) [reply app/templates/attendee-app.hbs]: @kushthedude I asked the paragraphs to be aligned in the mobile view , not completely removed :sweat_smile:

**Comment by kushthedude** [comment]
@uds5501 I took reference from other landing pages for mobile apps .

**Comment by kushthedude** [comment]
> Some of the Images are not optimized, 500KB is too big.

What must be the adequate size?

**Comment by kushthedude** [comment]
@CosmicCoder96 @pradeepgangwar I have reduced sizes as much as possible, Reducing them further will pixelate them, Please provide your reviews

**Comment by kushthedude** [comment]
@CosmicCoder96, Since I don't have `Scrolling SS` in my machine attaching two ss per page -:
Desktop View:

image
Screenshot 2019-05-22 at 4 54 06 PM

Mobile View -:
image
image

**Pull request merged**